### PR TITLE
feat: launch org-based billing

### PR DESCRIPTION
**Pull request by kevcodez**
- Adjust docs to contain hourly prices for compute
- Adjusts pricing page & compute pricing modal
- Team plan now fully self-serve, no longer links to the success form

### CODE REVIEW

File: studio/components/interfaces/Organization/GeneralSettings/MigrateOrganizationBillingButton.tsx [hunk 1]
```diff
@@ -108,9 +113,38 @@ const MigrateOrganizationBillingButton = observer(() => {
...
+              loading={!organization?.slug}
+              onClick={toggle}
+              type="primary"
+              disabled={!canMigrateOrg || disableOrgBillingMigration}
+            >
+              Migrate organization
+            </Button>
+          </Tooltip.Trigger>
+          {!canMigrateOrg ||
+            (disableOrgBillingMigration && (
```
Comment (alaister) [inline studio/components/interfaces/Organization/GeneralSettings/MigrateOrganizationBillingButton.tsx:128]: I think these parens are wrong? Should it be: `(!canMigrateOrg || disableOrgBillingMigration) &&`?

File: studio/components/interfaces/Settings/General/TransferProjectPanel/TransferProjectButton.tsx [hunk 2]
```diff
@@ -87,27 +88,34 @@ const TransferProjectButton = () => {
...
-              >
-                <span className="text-xs text-scale-1200">
-                  You need additional permissions to transfer this project
-                </span>
-              </div>
-            </Tooltip.Content>
-          </Tooltip.Portal>
-        )}
+        {!canTransferProject ||
+          (disableProjectTransfer && (
```
Comment (alaister) [inline studio/components/interfaces/Settings/General/TransferProjectPanel/TransferProjectButton.tsx:100]: Same here

File: studio/components/layouts/AppLayout/OrganizationDropdown.tsx [hunk 3]
```diff
@@ -97,13 +96,13 @@ const OrganizationDropdown = () => {
                 </ScrollArea>
               </CommandGroup_Shadcn_>
               <CommandGroup_Shadcn_ className="border-t">
-                <Link passHref href={orgCreationV2 ? `/new-with-subscription` : `/new`}>
+                <Link passHref href={`/new`}>
```
Comment (alaister) [inline studio/components/layouts/AppLayout/OrganizationDropdown.tsx:99]: very nit: `href="/new"`

**Review by alaister — COMMENTED** [review]
LGTM! Just a couple of code-style things:

File: studio/components/interfaces/Organization/GeneralSettings/MigrateOrganizationBillingButton.tsx (hunk 1 above)
Comment (kevcodez) [reply studio/components/interfaces/Organization/GeneralSettings/MigrateOrganizationBillingButton.tsx:128]: Good catch, fixing

File: studio/components/interfaces/Settings/General/TransferProjectPanel/TransferProjectButton.tsx (hunk 2 above)
Comment (kevcodez) [reply studio/components/interfaces/Settings/General/TransferProjectPanel/TransferProjectButton.tsx:100]: Good catch, fixing

**Review by alaister — APPROVED** [review]
LGTM!

File: apps/docs/pages/guides/platform/compute-add-ons.mdx [hunk 4]
```diff
@@ -8,21 +8,23 @@ export const meta = {
...
-| Large           | $100    | 2-core ARM (dedicated)  | 8 GB   | 160                 | 300                 |
-| XL              | $200    | 4-core ARM (dedicated)  | 16 GB  | 240                 | 700                 |
-| 2XL             | $400    | 8-core ARM (dedicated)  | 32 GB  | 380                 | 1500                |
-| 4XL             | $950    | 16-core ARM (dedicated) | 64 GB  | 480                 | 3000                |
-| 8XL             | $1,860  | 32-core ARM (dedicated) | 128 GB | 490                 | 6000                |
-| 12XL            | $2,790  | 48-core ARM (dedicated) | 192 GB | 500                 | 9000                |
-| 16XL            | $3,720  | 64-core ARM (dedicated) | 256 GB | 500                 | 12,000              |
+| Plan   | Hourly Price USD | Monthly Price USD | CPU                     | Memory | Connections: Direct | Connections: Pooler |
+| ------ | ---------------- | ----------------- | ----------------------- | ------ | ------------------- | ------------------- |
+| Stater | $0.01344         | ~$10              | 2-core ARM (shared)     | 1 GB   | 60                  | 200                 |
```
Comment (delgado3d) [inline apps/docs/pages/guides/platform/compute-add-ons.mdx:13]:
```suggestion
| Starter | $0.01344         | ~$10              | 2-core ARM (shared)     | 1 GB   | 60                  | 200                 |
```

Comment (kevcodez) [reply apps/docs/pages/guides/platform/compute-add-ons.mdx:13]: Whoops, thx

**Pull request merged**